### ISSUE TITLE
Cant install ecodev/graphql-upload on php 8

### ISSUE THREAD

**Issue by marhub** [thecodingmachine/graphqlite#364]
```
root@8a81bf3abbf8:/application# composer req ecodev/graphql-upload
Using version ^6.1 for ecodev/graphql-upload
./composer.json has been updated
Running composer update ecodev/graphql-upload
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires ecodev/graphql-upload ^6.1 -> satisfiable by ecodev/graphql-upload[6.1.0].
    - ecodev/graphql-upload 6.1.0 requires webonyx/graphql-php ^14.3 -> found webonyx/graphql-php[v14.3.0, ..., 14.x-dev] but the package is fixed to v0.13.9 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.

```

problem seems to be that webonyx is required on `^0.13.4`, on the other hand downgrading to `ecodev/graphql-upload` in version less than `6.1` won't work because 6.1 ist first one to accept PHP `v8`.

part of composer.json:

``` 
"require": {
        "php": ">=8.0.0",
        "ext-ctype": "*",
        "ext-iconv": "*",
        "beberlei/doctrineextensions": "^1.3",
        "beberlei/porpaginas": "^1.2",
        "composer/package-versions-deprecated": "*",
        "doctrine/doctrine-bundle": "^2.3",
        "doctrine/doctrine-migrations-bundle": "^3.1",
        "doctrine/orm": "^2.8",
        "ecodev/graphql-upload": "5",
        "fakerphp/faker": "^1.14",
        "lexik/jwt-authentication-bundle": "^2.11",
        "mockery/mockery": "^1.4",
        "moneyphp/money": "^4.0",
        "myclabs/php-enum": "^1.8",
        "nelmio/cors-bundle": "^2.1",
        "ramsey/uuid": "^4.1",
        "slevomat/coding-standard": "^7.0",
        "symfony/console": "5.2.*",
        "symfony/dotenv": "5.2.*",
        "symfony/flex": "^1.3.1",
        "symfony/framework-bundle": "5.2.*",
        "symfony/http-client": "5.2.*",
        "symfony/mailer": "5.2.*",
        "symfony/monolog-bundle": "^3.7",
        "symfony/proxy-manager-bridge": "5.2.*",
        "symfony/security-bundle": "5.2.*",
        "symfony/yaml": "5.2.*",
        "thecodingmachine/graphqlite-bundle": "^4.1",
        "yohang/finite": "^1.2"
    },
    ```

**Comment by marhub** [comment]
Also for anyone struggling, this seems to be quick fix just for now:
` "webonyx/graphql-php": "0.13.9 as 14.3.0",`

**Comment by oojacoboo** [comment]
Have you tried with `webonyx/graphql-php 14.3`?  There is a PR open (#362) to get the CI tests working.  Hopefully we can improve on compatibility going forward.

**Comment by marhub** [comment]
as i wrote in "quick fix" -> it seems to work with webonyx 14.3, but it cannot be done clean way because of https://github.com/thecodingmachine/graphqlite/blob/6b077e60fdef3ce53db3639de78d7d40fbb8f5d8/composer.json#L14

**Comment by oojacoboo** [comment]
Closing this issue.  This should be resolved by #283
